app/controllers/books_controller.rb

```diff
@@ -42,7 +42,9 @@ def create
   # PATCH/PUT /books/1.json
   def update
     respond_to do |format|
+      old_status = @book.status
       if @book.update(book_params)
+        check_for_reservation_changes(old_status, book_params[:status])
```

Comment:
How about we do this in an `after_update` callback?

Reply:
This method needs access to `current_user`, which is a [Clearance helper method](https://github.com/thoughtbot/clearance#helper-methods) only accessible from controllers, helpers and views, so I think we have to call it from here. Unless there's a way to call it as an `after_action` callback for #update -- but then, is there a way to do this with the required params?